pkg/operation/common/types.go

```diff
@@ -155,6 +155,10 @@ const (
 	// GardenRoleVpa is the value of GardenRole key indicating type 'vpa'.
 	GardenRoleVpa = "vpa"
 
+	// GardenControlPlaneMonitoring is the key for an annotation of a Shoot cluster whose value indicates if
+	// a monitoring stack for the shoot owner should be deployed.
+	GardenControlPlaneMonitoring = "shoot.garden.sapcloud.io/controlPlaneMonitoring"
```

Comment:
Can we go for `control-plane-monitoring` instead of camel case? We currently also have this e.g. for `GardenIgnoreAlerts`.

Reply:
done

Reply:
As already mentioned I would like to use an annotation like `monitoring.shoot.garden.sapcloud.io/enabled` instead of `shoot.garden.sapcloud.io/control-plane-monitoring`. See my comment above.

Reply:
Annotation should be `shoot.gardener.cloud/controlPlaneMonitoring`

Reply:
@mvladev why camel case? For our current annotations, it also isn't camel case.

Reply:
I meant that the domain should be `shoot.gardner.cloud`.
The annotation then should be `shoot.gardener.cloud/control-plane-monitoring`

Reply:
Ah yeah missed the domain part, good catch!